cleaner_engine.py

```
@@ -0,0 +1,1406 @@
+                        break
+                        
+        except Exception as e:
+            return f"Fehler bei Startup-Optimierung: {e}"
+        
+        return f"{disabled_count} verwaiste Startup-Einträge gefunden"
+    
+    async def _flush_dns_cache(self) -> str:
+        """Leert den DNS-Cache."""
+        try:
+            if sys.platform == "win32":
+                os.system("ipconfig /flushdns > nul 2>&1")
```

Ruff (0.12.2)

Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)